source/index.rst

```diff
@@ -79,17 +79,13 @@ Welcome to the FIRST Robotics Competition Documentation! This documentation will
 
 .. toctree::
    :maxdepth: 1
-   :caption: Contributing
+   :caption: Contributing to Documentation
 
    docs/contributing/contribution-guidelines
    docs/contributing/style-guide
    docs/contributing/build-instructions
    docs/contributing/top-contributors
+   Report a documentation issue <https://github.com/wpilibsuite/frc-docs/issues>
```

Comment:
This is be kept in it's own toctree to emphasis issue distinction, and the URL can be changed to point to the allwpilib repo, where we can move issues around as needed.